### DISCUSSION TITLE
How i can set custom Cache-Control header for document?

### DISCUSSION THREAD

**Discussion by greatUzumaki** [vercel/next.js#70251] [Help]
### Summary

I want to remove ```s-maxage``` in Cache control header, how i can do that?

![image](https://github.com/user-attachments/assets/f21a8760-0f9d-4989-8869-c92086fb514e)

### Additional information

I`m use Nextjs 14 with app dir

**Comment by leerob** [comment]
Are you using `revalidate` on this page?

**Reply by greatUzumaki** [reply]
No, but is it necessary?

This is all what i have on this page:

```
async function fetchData() {
...
}

const getData = unstable_cache(
  fetchData,
  [TAG],
  {
    tags: [TAG],
  },
);

export default async function Main() {
  const data = await getData();

  return <MainPage {...data} />;
}
```

**Reply by greatUzumaki** [reply]
I just want to set s-maxage/maxage=0

**Reply by leerob** [reply]
Interesting, why are you using `unstable_cache` then if you're wanting dynamic rendering? If you use a dynamic function, like reading the incoming request `searchParams`, or `cookies()`, that would make the page dynamically rendered.

**Reply by greatUzumaki** [reply]
I use unstable cache for redis, cause in fetch funtction i use axios, and i need tags for invalidating
